inst/shiny/DiagnosticsExplorer/ui.R

```diff
@@ -1238,18 +1238,20 @@ bodyTabItems <- shinydashboard::tabItems(
                                            )
                                          )
                               ),
-                              reactable::reactableOutput("compareCohortCharacterizationTable")),
+                              shinycssloaders::withSpinner(reactable::reactableOutput("compareCohortCharacterizationTable"))),
```

Comment:
i am fine with adding css loaded spinner , but i want to see how to interacts with progress messages

progress messages - as implemented tell us more than just loading a page. it tell if there is a database query going on vs is R doing a post processing of the data etc.